src/Microsoft.AspNetCore.DeveloperCertificates.Tools/Program.cs

```diff
@@ -0,0 +1,118 @@
+                });
+
+                app.HelpOption("-h|--help");
+
+                app.OnExecute(() =>
+                    {
+                        app.ShowHelp();
+                        return Success;
+                    });
+
+                return app.Execute(args);
+            }
+            catch
+            {
+                return CriticalError;
+            }
+        }
+
+        private static int EnsureHttpsCertificate(CommandOption exportPath, CommandOption password, CommandOption trust)
+        {
+            var now = DateTimeOffset.Now;
+            var manager = new CertificateManager();
+
+            var result = manager.EnsureAspNetCoreHttpsDevelopmentCertificate(
+                now,
+                now.Add(HttpsCertificateValidity),
+                exportPath.Value(),
+                trust == null ? false : trust.HasValue(),
+                password.HasValue(),
+                password.Value());
+
+            switch (result)
+            {
+                case EnsureCertificateResult.Succeeded:
+                    Console.WriteLine("The HTTPS developer certificate was generated successfully.");
+                    if (exportPath.Value() != null)
+                    {
+                        Console.WriteLine($"The certificate was exported to {Path.GetFullPath(exportPath.Value())}");
+                    }
+                    return Success;
+                case EnsureCertificateResult.ValidCertificatePresent:
+                    if (exportPath.Value() != null)
+                    {
+                        Console.WriteLine($"The certificate was exported to {Path.GetFullPath(exportPath.Value())}");
+                    }
+                    Console.WriteLine("A valid HTTPS certificate is already present.");
+                    return Success;
+                case EnsureCertificateResult.ErrorCreatingTheCertificate:
+                    Console.WriteLine("There was an error creating the HTTPS developer certificate.");
+                    return ErrorCreatingTheCertificate;
+                case EnsureCertificateResult.ErrorSavingTheCertificateIntoTheCurrentUserPersonalStore:
+                    Console.WriteLine("There was an error saving the HTTPS developer certificate to the current user personal certificate store.");
+                    return ErrorSavingTheCertificate;
+                case EnsureCertificateResult.ErrorExportingTheCertificate:
+                    Console.WriteLine("There was an error exporting HTTPS developer certificate to a file.");
+                    return ErrorExportingTheCertificate;
+                case EnsureCertificateResult.FailedToTrustTheCertificate:
+                    Console.WriteLine("There was an error trusting HTTPS developer certificate.");
+                    return ErrorTrustingTheCertificate;
+                default:
+                    Console.WriteLine("Something went wrong. The HTTPS developer certificate could not be created.");
```

Comment:
Switch error output to `Console.Error.WriteLine`. Also, consider using `IReporter.Error`, like the other cli tools from this repo.